src/index.ts

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env ts-node
 'use strict';
 
+console.log('test DOP-3596');
```

Comment:
```suggestion
```

Reply:
woops! remnant from testing GH action to see if it'd commit gitignored files